aten/src/ATen/nn.yaml

```diff
@@ -131,6 +131,7 @@
   scalar_check:
     output: 'false'
     grad_input: 'false'
+  scalar_types: ['Float', 'Double', 'Half', 'Long']
```

Comment:
can we support cpu_scalar_types and cuda_scalar_types since we don't actually support conv2d for longs on CUDA anyway?

Reply:
Added implementation: https://github.com/pytorch/pytorch/pull/20370